test/Microsoft.AspNet.StaticFiles.Tests/StaticFileMiddlewareTests.cs

```diff
@@ -31,6 +31,18 @@ public class StaticFileMiddlewareTests
             Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
         }
 
+        [Fact]
+        public async Task InvalidCharacters_PassesThrough()
+        {
+            TestServer server = StaticFilesTestServer.Create(app => app.UseStaticFiles(new StaticFileOptions()
+            {
+                RequestPath = new PathString(""),
+                FileProvider = new PhysicalFileProvider(Path.Combine(Directory.GetCurrentDirectory(), "."))
```

Comment:
Whats the point of this:

`Path.Combine(Directory.GetCurrentDirectory(), ".")`

Why not just do `Directory.GetCurrentDirectory()`